pkg/apis/options/upstreams.go

```diff
@@ -49,12 +49,13 @@ type Upstream struct {
 	// Defaults to 1 second.
 	FlushInterval *time.Duration `json:"flushInterval,omitempty"`
 
-	// PassHostHeader determines whether the request host header should be proxied
-	// to the upstream server.
-	// Defaults to true.
-	PassHostHeader bool `json:"passHostHeader"`
-
-	// ProxyWebSockets enables proxying of websockets to upstream servers
-	// Defaults to true.
-	ProxyWebSockets bool `json:"proxyWebSockets"`
+	// SkipHostHeader determines whether the request host header should be proxied
+	// to the upstream server. The host header is passed by default.
+	// Defaults to false.
+	SkipHostHeader bool `json:"passHostHeader"`
```

Comment:
Should we leave this as `Pass`? In other areas of the codebase we've used the word Pass in alot of cases to specify headers to the upstream (which this matches). Where `skip` has overwhelmingly been used in the context of skipping authentication.

Reply:
Hmm, that's a good point. Do you think we should go down the pointer to bool route instead? That would allow us to default to true if it is not set. 

I'd prefer not to change the default if we don't have good reason, but we need boolean options to default to `false` or be bool pointers

Reply:
Ok, I switched the names back here and moved them to pointers, it's not too bad, most harm is done during the testing 😅

Reply:
Forgot about the `false` defaults in the struct and `*bool` needed to get a true default 🤦 

And here I thought I made an innocuous naming pattern comment.